desktop/renderer/index.js

```diff
@@ -21,7 +21,8 @@ import {setupContextMenu} from '../app/menu-helper'
 import electron, {ipcRenderer} from 'electron'
 import RemoteManager from './remote-manager'
 import {ipcLogsRenderer} from '../app/console-helper'
-import _ from 'lodash'
+import loadPerf from '../shared/util/load-perf'
+import merge from 'lodash/merge'
```

Comment:
we can include just the sub things we need. won't matter when/if we add treeshaking